### ISSUE TITLE
image_picker: Press lock button within application leads to auto lockscreen on start camera

### ISSUE THREAD

**Issue by addyu** [flutter/flutter#94361]
After pressing the lock screen button and resume within our application using image_picker package, starting the camera leads the screen to be locked again. 

https://user-images.githubusercontent.com/32528643/143922733-b0923f7d-ec8f-4829-b420-cda3e69c4eda.mp4

There is no exception happening. Also flutter doctor does not output any errors.

Tested on Android 10, 11 and 12.

**Comment by mamuseferha** [comment]
Hi @addyu 
Can you please provide your `flutter doctor -v` and a [minimal complete reproducible code sample](https://stackoverflow.com/help/minimal-reproducible-example)?
Thank you

**Comment by addyu** [comment]
Found the problem.
In AndroidManifest.xml if 

`<activity
    android:showWhenLocked="true"
    android:turnScreenOn="true"
    ...
`
is set, pushing the lock button blocks the lock screen to appear. As I switched to the camera with image_picker, this block seems to come off and the lock screen turns on.

Not an issue of this plugin I guess (attributes were set, because of this plugin: https://pub.dev/packages/flutter_local_notifications).

Thank you for the fast reply.
